appservice/src/SiteWrapper.ts

```diff
@@ -183,24 +183,16 @@ export class SiteWrapper {
 
     public async deploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, configurationSectionName: string, confirmDeployment: boolean = true): Promise<void> {
         const config: SiteConfigResource = await this.getSiteConfig(client);
-        const kuduClient: KuduClient = await this.getKuduClient(client);
-        const deploymentId: string = 'latest';
-        let previousDeploymentId: string | undefined;
-        try {
-            previousDeploymentId = (await kuduClient.deployment.getResult(deploymentId)).id;
-            // current active deployment used to verify new deployment was successful
-        } catch (error) {
-            // swallow the error-- getResult('latest') should only fail if this is the first deployment
-        }
+        const deploySendTime: Date = new Date();
```

Comment:
I don't think you should be comparing local machine time with server time. That's what this does, right?